paas_app_charmer/charm.py

```diff
@@ -375,12 +382,12 @@ def _on_update_status(self, _: ops.HookEvent) -> None:
     @block_if_invalid_config
     def _on_mysql_database_database_created(self, _: DatabaseRequiresEvent) -> None:
         """Handle mysql's database-created event."""
-        self.restart()
+        self.restart(rerun_migrations=True)
```

Comment:
only one of database_created or enpoints_changed is called by the data library, so it will not run twice when the charm is integrated with the database.